src/PrestaShopBundle/Controller/Admin/ModuleController.php

```diff
@@ -355,13 +357,46 @@ public function configureModuleAction($module_name)
     {
         /* @var $legacyUrlGenerator UrlGeneratorInterface */
         $legacyUrlGenerator = $this->get('prestashop.core.admin.url_generator_legacy');
+        $legacyContextProvider = $this->get('prestashop.adapter.legacy.context');
+        $legacyContext = $legacyContextProvider->getContext();
+        $moduleManagerFactory = new ModuleManagerBuilder();
+        $moduleRepository = $moduleManagerFactory->buildRepository();
+        // Get accessed module object
+        $moduleAccessed = $moduleRepository->getModule($module_name);
+
+        // Get current employee ID
+        $currentEmployeeID = $legacyContext->employee->id;
+        // Get accessed module DB ID
+        $moduleAccessedID = (int)$moduleAccessed->database->get('id');
+
+        // Save history for this module
```

Comment:
all this job should be done in a service, how about a `moduleHistoryManager` class with a méthode `logAccess(employeeId, moduleId)` ?